### ISSUE TITLE
Plan work to apply Melody designs to existing components

### ISSUE THREAD

**Issue by wintondeshong** [AndcultureCode/AndcultureCode.JavaScript.React.Components#40]
A ways back the andculture design team created a design system called "Melody". The team would like to leverage that work and use it as our base styles for this component library.

There will be several components that were never a part of the Melody designs so we will need to collaborate with the design team on those.

Resources:
- Invision: https://andculture.invisionapp.com/d/main#/projects/prototypes/13112332
- TODO: Get complete set of design files that can be shared publically (pending)

**Comment by TheDataDesigner** [comment]
@wintondeshong Is this referring to the library UI (like the sidebar, header, etc) or the component styles themselves? The component styles shown on that Invision board are actually from Fathom, which wasn't intended to be base styles for any component library.

I had started implementing a default style system into our version of the component library that could be of use here. It was designed to be minimal and easily overridden, either by changing variables within a global "design spec" scss file or by defining styles on a per-component basis. This allows global styles to be changed quickly and easily project-wide, but also have total control to override anything when needed.

Since I designed Fathom and the original Melody system, I'd be happy to take the lead on getting global default styles set up.

**Comment by wintondeshong** [comment]
> @wintondeshong Is this referring to the library UI (like the sidebar, header, etc) or the component styles themselves? The component styles shown on that Invision board are actually from Fathom, which wasn't intended to be base styles for any component library (at least when I originally designed it).
> 
> I had started implementing a default style system into our version of the component library that could be of use here. It was designed to be minimal and easily overridden, either by changing variables within a global "design spec" scss file or by defining styles on a per-component basis. This allows global styles to be changed quickly and easily project-wide, but also have total control to override anything when needed.

Yeah, calling out the use of Melody was an initial misunderstanding. Brad brought up that what we were referring to was really only a style to the actual hud/UI for the component library (in this case storybook). 

As for next steps, it would be great to get a base component style created so we could have a great jumping off point on projects while final designs are being created and implemented.
